project2/styles.css

```diff
@@ -0,0 +1,119 @@
+* {
+    box-sizing: border-box;
+}
+
+body {
+    background-color: rgb(255, 255, 255);
+    color: rgb(0, 0, 0);
+    font-family: 'Be Vietnam', sans-serif;
+}
+
+section {
+    display: inline-block;
+    margin: auto;
+    padding-bottom: 15px;
+    width: 400px;
+
+}
+
+h2,
+ul,
+p {
+    padding-left: 5px;
+    padding-right: 5px;
+}
+
+a:link,
+a {
+    color: black;
+    text-decoration: none;
+}
+
+a+a {
+    margin-left: 5px;
+}
+
+h2 {
+    font-size: 24px;
+    font-weight: 800;
+    text-decoration: none;
```

Comment:
This rule isn't doing anything -- the `text-decoration` is effective on links (`a`), which are the only elements that have default `text-decoration`. Adding it on the parent element (this `h2`) doesn't do anything:
```suggestion
```